### DISCUSSION TITLE
How to use gist or embed codes in discussion?

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#9440] [Discussions]
Selâm (hi)

Tartışmada öz veya gömme kodları nasıl kullanılır? (How to use gist or embed codes in discussion?)

Örnek için (For example);
<script src="https://gist.github.com/my_user_name/?????????????.js"></script>

**Comment by turkmenistanli** [comment]
Bu önerinin uygulanması yazılım dışındakilerin daha kolay githup'a katılımını sağlar.
